spec/specs_spec.rb

```diff
@@ -293,6 +293,11 @@ def get_true_hash_size(hash, obj_class)
     $dut.has_spec?(:tnikhov2).should == true
     $dut.has_spec?(/tnikhov/).should == true
   end
+
+  it "finding specs via :symbol option works" do
+    $dut.has_spec?(nil, symbol: :gvdd).should == true  
```

Comment:
I think this would be more idiomatic Origen:

~~~ruby
$dut.has_spec?(:gvdd, symbol: true)
~~~

What do you think?